src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -12,8 +12,8 @@
     <!-- CS1698 - Disable warning about reference to 4.0.0.0 System.Runtime.WindowsRuntime having same simple name as target assembly -->
     <NoWarn>$(NoWarn)1698</NoWarn>
     <ProjectGuid>{844A2A0B-4169-49C3-B367-AFDC4894E487}</ProjectGuid>
-    <TargetPackageFramework Condition="'$(TargetPackageFramework)'==''">dotnet5.3</TargetPackageFramework>
-    <TargetPackageRuntime>win8</TargetPackageRuntime>
+    <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">dotnet5.4</PackageTargetFramework> 
+    <PackageTargetRuntime>win8</PackageTargetRuntime>
     <AssemblyVersion>4.0.11.0</AssemblyVersion>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETPlatform,Version=v5.3</NuGetTargetMoniker>
```

Comment:
PackageTargetFramework and NuGetTargetMoniker should not be out of sync when using dotnet/netplatform.